packages/blazor-workspace/NimbleBlazor/Components/NimbleWaferMap.razor.cs

```diff
@@ -118,6 +123,23 @@ public partial class NimbleWaferMap : ComponentBase
         }
     }
 
+    /// <summary>
+    /// Represents the input data, an apache arrow Table, which fills the wafer map with content
+    /// </summary>
+    [Parameter]
+    public RecordBatch? DiesTable
```

Comment:
What's the reason for using `RecordBatch` here instead of the Apache `Table` type?

Reply:
The [ArrowStreamWriter](https://github.com/apache/arrow/blob/main/csharp/src/Apache.Arrow/Ipc/ArrowStreamWriter.cs) does not support Tables from what I understand